### PR TITLE
fix: update log event origins

### PR DESCRIPTION
**Pull request by omBratteng**
## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://as-788-fix-origin.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/layout/common.tsx [hunk 1]
```diff
@@ -132,7 +133,12 @@ export const SearchControlHeader = ({
     ) : null,
     showPlusSubscription &&
     feedsWithActions.includes(feedName as SharedFeedPage) ? (
-      <ToggleClickbaitShield key="toggle-clickbait-shield" />
+      <ToggleClickbaitShield
+        origin={
+          feedName === SharedFeedPage.Custom ? Origin.CustomFeed : Origin.Feed
```
Comment (omBratteng) [inline packages/shared/src/components/layout/common.tsx:138]: `Origin.Feed` is a little bit vague as it's used on multiple feeds, but there's no `my-feed` origin at the moment and unsure if I should add it @idoshamun?

Comment (rebelchris) [reply packages/shared/src/components/layout/common.tsx:138]: I think we use `feedName` elsewhere for this matter.
You can get it from activeFeedContext if I remember correct, there should be examples of this origin already.

Maybe check taxonomy for one that returns it, and we can mimic that.
Maybe postAnalyticsEvent should have it, 99% sure.

Comment (rebelchris) [reply packages/shared/src/components/layout/common.tsx:138]: @omBratteng We good here, I somehow thought this was on multiple feeds, but it's only my-feed and custom so this works better.

**Review by rebelchris — APPROVED** [review]
Good to go

**Pull request merged**